site/src/examples/scrim/WithChild.tsx

```diff
@@ -0,0 +1,110 @@
+import { ReactElement, useState } from "react";
+import {
+  Button,
+  Card,
+  Display3,
+  H3,
+  Scrim,
+  Spinner,
+  StackLayout,
+  Text,
+} from "@salt-ds/core";
+import { ArrowDownIcon, ArrowUpIcon } from "@salt-ds/icons";
+
+export const WithChild = (): ReactElement => {
+  const [open, setOpen] = useState(false);
+  const [buttonText, setButtonText] = useState("Show scrim");
+
+  const handleClick = () => {
+    setOpen(!open);
+    setButtonText(!open ? "Hide scrim" : "Show scrim");
```

Comment:
Nit: we shouldn't use state for stuff like this. https://react.dev/learn/you-might-not-need-an-effect#updating-state-based-on-props-or-state

It can be derived from state, so where buttonText is used should just be {!open ? "Hide scrim" : "Show scrim"}